tests/metrics_test.py

```diff
@@ -1846,381 +1692,198 @@ def test_as_flat_list(self):
-            }
-        }, select(metric_list, "throughput", operation="index"))
-
-        self.assertEqual({
-            "name": "service_time",
-            "task": "index #1",
-            "operation": "index",
-            "value": {
-                "50": 341,
-                "100": 376
+            select(metric_list, "throughput", operation="index"),
+        )
+
+        self.assertEqual(
+            {
+                "name": "service_time",
+                "task": "index #1",
+                "operation": "index",
+                "value": {"50": 341, "100": 376},
+                "meta": {"clients": 8, "phase": "idx"},
             },
-            "meta": {
-                "clients": 8,
-                "phase": "idx"
-            }
-        }, select(metric_list, "service_time", operation="index"))
-
-        self.assertEqual({
-            "name": "latency",
-            "task": "index #1",
-            "operation": "index",
-            "value": {
-                "50": 340,
-                "100": 376
+            select(metric_list, "service_time", operation="index"),
+        )
+
+        self.assertEqual(
+            {
+                "name": "latency",
+                "task": "index #1",
+                "operation": "index",
+                "value": {"50": 340, "100": 376},
+                "meta": {"clients": 8, "phase": "idx"},
             },
-            "meta": {
-                "clients": 8,
-                "phase": "idx"
-            }
-        }, select(metric_list, "latency", operation="index"))
-
-        self.assertEqual({
-            "name": "error_rate",
-            "task": "index #1",
-            "operation": "index",
-            "value": {
-                "single": 0.0
+            select(metric_list, "latency", operation="index"),
+        )
+
+        self.assertEqual(
+            {
+                "name": "error_rate",
+                "task": "index #1",
+                "operation": "index",
+                "value": {"single": 0.0},
+                "meta": {"clients": 8, "phase": "idx"},
             },
-            "meta": {
-                "clients": 8,
-                "phase": "idx"
-            }
-        }, select(metric_list, "error_rate", operation="index"))
-
-        self.assertEqual({
-            "name": "throughput",
-            "task": "search #2",
-            "operation": "search",
-            "value": {
-                "min": 9,
-                "mean": 10,
-                "median": 10,
-                "max": 12,
-                "unit": "ops/s"
-            }
-        }, select(metric_list, "throughput", operation="search"))
-
-        self.assertEqual({
-            "name": "service_time",
-            "task": "search #2",
-            "operation": "search",
-            "value": {
-                "50": 98,
-                "100": 110
-            }
-        }, select(metric_list, "service_time", operation="search"))
-
-        self.assertEqual({
```

Comment:
Can we please keep the current formatting here? I'll stop my individual comments at this point but this applies similarly to all nested structures below.